modules/ossm-security-mtls.adoc

```diff
@@ -61,3 +61,26 @@ spec:
     tls:
       mode: ISTIO_MUTUAL
 ----
+
```

Comment:
@knrc Here's the draft content for https://issues.redhat.com/browse/MAISTRA-1456